README.md

```diff
@@ -4,7 +4,13 @@
 ![GitHub branch checks state](https://img.shields.io/github/checks-status/opf/openproject/dev)
 [![Github Tests](https://github.com/opf/openproject/actions/workflows/test-core.yml/badge.svg?branch=dev)](https://github.com/opf/openproject/actions/workflows/test-core.yml)
 
-OpenProject is a web-based project management software. Its key features are:
+**We empower teams to achieve great things together for the good of society.**
+
+OpenProject is a web-based project management software. Use OpenProject to manage your projects, tasks and goals. Collaborate via work packages and link them to your pull requests on Github. [Read more about the OpenProject GitHub integration](https://www.openproject.org/docs/system-admin-guide/integrations/github-integration/).
+
+![Screenshot of OpenProject, showing the GitHub tab on a work package](GitHub-tab.png)
```

Comment:
I think we should update the screenshot for a newer one (button color and the color of the frame around GitHub tab have been updated since this one was taken it seems)